packages/integration-tests/tsconfig.json

```diff
@@ -1,12 +1,22 @@
 {
   "extends": "../../tsconfig.base.json",
-  "compilerOptions": { "rootDir": "src", "outDir": "lib", "allowJs": true },
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "lib",
+    "allowJs": true,
+    "paths": {
+      "@env/*": [
+        "./src/test-projects/data-storage-auth-with-triggers-ts/.amplify/function-env/*"
+      ]
+    }
+  },
   "references": [
     { "path": "../auth-construct" },
     { "path": "../backend" },
     { "path": "../backend-secret" },
     { "path": "../client-config" },
     { "path": "../platform-core" }
   ],
+  "include": ["src/**/*", "src/**/.amplify/**/*"],
```

Comment:
Without this Typescript will complain that the type definition file is not listed within the file list of the project. Also for some reason `src/**/*` does not include the `.amplify` folder so I had to explicitly include a path with it.